Documentation/sourcebuild-user-onboarding-guide/prerequisites.md

```diff
@@ -0,0 +1,15 @@
+## Build Requirements and Prerequisites
+
+Please click on the link above that matches your machine and ensure you have installed all the prerequisites for the build to work.  
+
+|**Chip** | **Linux**  |**Windows** |**MacOS**|
+|:--------------:|:----------------:|:--------------:|:----------------:|
+|x64|Supported |Not Supported|Not Supported|
+|x86|Not Supported |Not Supported|Not Supported|
+|ARM64|Supported |Not Supported|Not Supported|
+||[Prerequisites](https://github.com/dotnet/runtime/blob/main/docs/workflow/requirements/linux-requirements.md)|||
+
+Additionally, keep in mind that cloning takes roughly ? MB of network transfer, inflating to a repository that can consume somewhere between ? GB. A build of the repo can take somewhere between ? GB of space for a single OS and Platform configuration. This might increase over time, so consider this to be a minimum bar for working with this codebase.
```

Comment:
```suggestion
Additionally, keep in mind that cloning takes roughly 1.5GB of network transfer, inflating to a repository that can consume somewhere around 600 MB. A build of the repo can take somewhere around 80 GB of space for a single OS and Platform configuration. This might increase over time, so consider this to be a minimum bar for working with this codebase.
```

Runtime is at least 1GB of transfer on its own.  On-disk immediately after cloning and doing `ArcadeBuildTarball` is about 600MB I think.  7.0 builds are about 60-80 GB I believe.  Can confirm these numbers after I'm back.

Reply:
Added the approximate numbers suggested in here. I will update as require.